packages/vats/src/proposals/resetRoundIds.js

```diff
@@ -0,0 +1,50 @@
+import { E } from '@endo/far';
+import { makeStorageNodeChild } from '@agoric/internal/src/lib-chainStorage.js';
+import { makeMarshal } from '@endo/marshal';
+import { Fail } from '@agoric/assert';
+
+/**
+ * @param {BootstrapPowers & {
+ *   consume: { chainStorage: any };
+ * }} powers
+ * @param {object} options
+ * @param {{ tokenName: string }} options.options
+ */
+export const resetRoundId = async (
+  { consume: { chainStorage, chainTimerService } },
+  options,
+) => {
+  const tokenName = options.options;
+  const priceFeedStorageNode = await makeStorageNodeChild(
+    chainStorage,
+    'priceFeed',
+  );
+  const tokenStorageNode = await makeStorageNodeChild(
+    priceFeedStorageNode,
+    `${tokenName}-USD_price_feed`,
+  );
+  const latestRoundStorageNode = await makeStorageNodeChild(
+    tokenStorageNode,
+    'latestRound',
+  );
+  const marshalData = makeMarshal(_val => Fail`data only`);
```

Comment:
Getting a failure here, I believe because of the timer's brand

Reply:
Found where to pickup a better marshaller.